GraphBLAS/cmake_modules/SuiteSparsePolicy.cmake

```diff
@@ -155,7 +155,7 @@ endif ( )
 # find this one without "make install"
 list ( FIND CMAKE_BUILD_RPATH ${CMAKE_BINARY_DIR} _idx )
 if ( _idx LESS 0 )
-    # not yet included in CMAKE_INSTALL_RPATH
+    # not yet included in CMAKE_BUILD_RPATH
```

Comment:
Notice that CMake always sets rpaths during the build, so that just-built executables can be run. I might be missing something, but I think this bit may be redundant.

It's on install that they are cleared by default.

Reply:
Checking the commit history, the CMAKE_BUILD_RPATH was set since the very first version of `SuiteSparsePolicy.cmake` (and I never thought about it).

But iiuc, you are right: It should not be necessary to set it.

For a test, I've just removed it. Let's see if something breaks without it in CI.

Reply:
The CI passed all tests without issues. 👍